tests/autoyast/prepare_profile.pm

```diff
@@ -32,6 +33,12 @@ sub run {
     }
     # Upload modified profile
     save_tmp_file($path, $profile);
+    # Copy profile to ulogs directory, so profile is available in job logs
+    # Works only with QEMU backend
```

Comment:
Why only for qemu?

Reply:
I've discussed it with @foursixnine, he told me that this trick will work only with qemu backend

Reply:
Currently for ay we use only qemu backend, so I've removed check. Let's see if it works for svirt once we enable tests. I've failed to get the run for shared s390x worker in 20 minutes, so I guess it doesn't make much sense to spend more time on this, but let it fail (if at all).